code/modules/mob/living/carbon/xenomorph/egg.dm

```diff
@@ -0,0 +1,202 @@
+/obj/effect/alien/egg
```

Comment:
i...wonder what's the thing about this being a child of effect rather than a structure or something like that ?

Reply:
I dunno, all alien things are like and i see no reason to change it